gen/suites/actor_creation/params.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/specs-actors/actors/abi"
+	"github.com/filecoin-project/specs-actors/actors/abi/big"
+	"github.com/filecoin-project/specs-actors/actors/runtime/exitcode"
+
+	. "github.com/filecoin-project/test-vectors/gen/builders"
+)
+
+func createActorUnparsableParams(v *Builder) {
+	v.Messages.SetDefaults(GasLimit(1_000_000_000), GasPremium(1), GasFeeCap(200))
+
+	sender := v.Actors.Account(address.SECP256K1, abi.NewTokenAmount(1_000_000_000_000))
+	receiver := v.Actors.Account(address.SECP256K1, abi.NewTokenAmount(1_000_000_000_000))
+
+	v.CommitPreconditions()
+
+	balanceBefore := v.Actors.Balance(sender.Robust)
+
+	// Valid message for construction of a payment channel
+	createMsg := v.Messages.Sugar().CreatePaychActor(sender.Robust, receiver.Robust, Value(abi.NewTokenAmount(10_000)), Nonce(0))
```

Comment:
Aside: I can't tell by inspection that this message is indeed valid. Does the builder have a means to commit this message to demonstrate its success, and then rollback to commit the busted version against the same prior state?

Reply:
Yeah, good point; we need a control test case here. I think having a separate test case that uses a valid address and a valid actor CID should be enough.

Reply:
Added in: https://github.com/filecoin-project/test-vectors/pull/41/commits/1af4b4dc89f48a829ed558a75e22ae51ca293957.

Reply:
And you were right, there were some bugs that were exercising a different path:
1. we were using robust addresses instead of ID addresses; but we weren't adding a mapping to the init actor (out of scope for this test), therefore the internal addr resolution failed. I've switched to ID addresses, and now the correct code paths are exercised.
2. we weren't validating the caller in the chaos actor, which wasn't a problem for failure cases because the `CreateActor` failed before returning, but it became a problem for the OK case.

Learning: we should ensure that all failure cases are accompanied by control cases. Also, when surgically targeting a particular code path, use step-through debugging to confirm that the right lines are being hit.